src/main/java/nextstep/subway/favorite/ui/FavoriteController.java

```diff
@@ -18,22 +20,22 @@ public FavoriteController(FavoriteService favoriteService) {
     }
 
     @PostMapping("/favorites")
-    public ResponseEntity createFavorite(@RequestBody FavoriteRequest request) {
-        favoriteService.createFavorite(request);
+    public ResponseEntity<?> createFavorite(@RequestBody FavoriteRequest request, @AuthenticationPrincipal UserDetail userDetail) {
```

Comment:
ResponseEntity의 body가 null임을 강제하고 싶다면 Void를 활용해도 좋습니다
```suggestion
    public ResponseEntity<Void> createFavorite(@RequestBody FavoriteRequest request, @AuthenticationPrincipal UserDetail userDetail) {
```